bundles/CoreBundle/src/Migrations/Version20230111074323.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Model\Tool\SettingsStore;
+
+/**
+ * Granting Word export permission for translation permission!
+ */
+final class Version20230111074323 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Grant Word export permission to those who already had "translation" permission, due Word export functionalities being moved out to be Standalone from Core Translation';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // Append to the comma separated list whenever the permissions text field has 'translation' but not already word_export
+        $this->addSql("INSERT INTO `users_permission_definitions` (`key`, `category`) VALUES ('word_export', 'Pimcore Word Export Bundle')");
+
+        $this->addSql('UPDATE `users` SET `permissions`=CONCAT(`permissions`, \',word_export\') WHERE `permissions` REGEXP \'(?:|,)translations(?:|,)\'');
```

Comment:
This REGEX needs to be a little more stricter, it could potentially trigger in these unwanted cases  eg. "someotherbundle_translations" or "translations_interface"
https://regex101.com/r/Q3TYnW/1

Reply:
You are right, but I am not sure how to handle this. 
Since these are the combinations how it could appear in the permissions:
translations
translations,
,translations
,translations,

Do you have any suggestions?